tmt/base.py

```diff
@@ -743,6 +761,9 @@ def go(self):
         self.debug('environment', self.environment, 'magenta', level=3)
         self.debug('context', self._fmf_context(), 'magenta', level=3)
 
+        # load environment into the shell
+        os.environ.update(self.environment)
```

Comment:
Why it modifies tmt process' environment? IMHO we should not touch it and just propagate self.environment to respective calls.

Reply:
Agreed. I'm not sure why the tmt process environment should be modified. If we just make sure that the environment is correctly handed over to `guest.execute()` including local provision implementation the executed commands such as `ansible-playbook` should get all that they need.

Reply:
okay